src/fields/optional/fieldCleave.vue

```diff
@@ -58,8 +65,20 @@ export default {
 			}
 		});
 	},
-
+	watch: {
+		value(formattedValue) {
+			if (this.cleave) {
+				this.$nextTick(() => {
+					// nextTick here is needed to allow Cleave to render the correct formatted value to be saved
+					this.cleave.setRawValue(formattedValue);
```

Comment:
Does setting this raw value trigger the listener to call `inputChange` which will trim the value? If not, we might want to call `this.inputChange()` after setting the raw value but that might cause a recursive loop because that function in turn sets `this.value`

Reply:
https://github.com/nosir/cleave.js/blob/e3fa6f3637fa6d8ea97ef66ac1957950632b72ab/src/Cleave.js#L421
-> setRawValue emits `input` event.